libvast/src/system/indexer.cpp

```diff
@@ -69,14 +78,30 @@ caf::behavior indexer(caf::stateful_actor<indexer_state>* self, type index_type,
           self->quit(err);
         });
     },
+    // FIXME: Get rid of one of these handler, i think the bottom one is
+    // currently unused.
```

Comment:
Can we get rid of it? Maybe that happens later in this PR.

Reply:
It does, I've squashed all related commits so the history shouldnt contain any of these anymore.